dev_guide/pod_autoscaling.adoc

```diff
@@ -212,21 +213,155 @@ $ oc create -f hpa.yaml
+** *Minpods/Maxpods*. The minumum and maximum number of replicas that can be set by the autoscaler
 
-$ oc describe hpa/frontend
-Name:                           frontend
+* Use the `oc describe` command for detailed information on the horizontal pod autoscaler object. 
++
+----
+$ oc describe hpa/hpa-resource-metrics-cpu
+Name:                           hpa-resource-metrics-cpu
 Namespace:                      default
 Labels:                         <none>
 CreationTimestamp:              Mon, 26 Oct 2015 21:13:47 -0400
 Reference:                      DeploymentConfig/default/frontend/scale
-Target CPU utilization:         80%
-Current CPU utilization:        79%
-Min pods:                       1
-Max pods:                       10
+Target CPU utilization:         80% <1>
+Current CPU utilization:        79% <2>
+Min replicas:                   1 <3>
+Max replicas:                   4 <4>
+ReplicationController pods:     1 current / 1 desired
+Conditions: <4>
+  Type                  Status  Reason                  Message
+  ----                  ------  ------                  -------
+  AbleToScale           True    ReadyForNewScale        the last scale time was sufficiently old as to warrant a new scale
+  ScalingActive         True    ValidMetricFound        the HPA was able to succesfully calculate a replica count from pods metric http_requests
+  ScalingLimited        False   DesiredWithinRange      the desired replica count is within the acceptible range
+Events:
 ----
-====
++
+<1> The average percentage of the requested memory that each pod should be using.
+<2> The current CPU utilization across all pods controlled by the deployment configuration.
+<3> The minimum number of replicas to which to scale down.
+<4> The maximum number of replicas to which to scale up.
+<4> If the object used the `v2alpha1` API, xref:viewing-a-hpa-status[status conditions] are displayed. 
+
+[[viewing-a-hpa-status]]
+=== Viewing Horizontal Pod Autoscaler Status Conditions
+
+You can use the status conditions set to determine  
+whether or not the horizontal pod autoscaler is able to scale and whether or not it is currently restricted
+in any way.
+
+The following status conditions are set:  
+
+* `AbleToScale` indicates whether the horzizontal pod autoscaler is able to fetch and update scales, and
+whether any backoff-related conditions would prevent scaling.
+** A `True` condition indicates scaling is allowed.
+** A `False` condition indicates scaling is not allowed for the reason specified.  
+* `ScalingActive` indicates whether the horzizontal pod autoscaler is enabled (the replica count of the target is not zero) and
+is able to calculate desired scales. 
+** A `True` condition indicates metrics is working properly.
+** A `False` condition generally indicates {product-title} is having problems with
+fetching metrics.   
+* `ScalingLimited` indicates that autoscaling is not allowed because a maximum or minimum quota was reached.  
```

Comment:
I would avoid the term "quota" here, since it has a very specific meaning in Kubernetes.  Just use "replica count" or something similar like you do a sentence later on.